.gitmodules

```diff
@@ -121,3 +118,6 @@
 [submodule "vendor/status-keycard-go"]
 	path = vendor/status-keycard-go
 	url = https://github.com/status-im/status-keycard-go.git
+[submodule "vendor/SortFilterProxyModel"]
```

Comment:
Do we have this submodule twice now?

Reply:
It was only in StatusQ, seems to be ok.

Reply:
Yeah, both `DOtherSide` lib and `sandbox` app need `SortFilterProxyModel` package. To avoid duplication we could make `sandbox` dependent on `vendor/DOtherSide/vendor/SortFilterProxyModel` but that is not the cleanest solution. I would leave it as is.

Reply:
yeah, I also would leave it as it is. `DOtherSide` is a separate thing and we shouldn't rely on dependencies from this submodule.